tests/bin/bats-exec-test

```diff
@@ -66,11 +66,12 @@ run() {
 }
 
 setup() {
-  true
+  export TEST_DIR=$(mktemp -d)
 }
 
 teardown() {
-  true
+  rm -f ${TEST_DIR}/*
```

Comment:
Could we maybe double check $TEST_DIR is set and non-empty for peace of mind?

Reply:
Done.